src/gui/Setup.qml

```diff
@@ -0,0 +1,147 @@
+        x: 234 * virtualstudio.uiScale; y: 200 * virtualstudio.uiScale
+        width: parent.width - x - (16 * virtualstudio.uiScale); height: 36 * virtualstudio.uiScale
+        visible: virtualstudio.selectableBackend
+    }
+    
+    Text {
+        id: backendLabel
+         anchors.verticalCenter: backendCombo.verticalCenter
+        x: leftMargin * virtualstudio.uiScale
+        text: "Audio Backend"
+        font { family: "Poppins"; pixelSize: fontMedium * virtualstudio.fontScale * virtualstudio.uiScale }
+        visible: virtualstudio.selectableBackend
+    }
+    
+    Text {
+        id: jackLabel
+        x: leftMargin * virtualstudio.uiScale; y: 200 * virtualstudio.uiScale
+        width: parent.width - x - (16 * virtualstudio.uiScale)
+        text: "Using JACK for audio input and output. Use QjackCtl to adjust your sample rate, buffer, and device settings."
+        font { family: "Poppins"; pixelSize: fontMedium * virtualstudio.fontScale * virtualstudio.uiScale }
+        wrapMode: Text.WordWrap
+        visible: virtualstudio.audioBackend == "JACK" && !virtualstudio.selectableBackend
+    }
+    
+    ComboBox {
+        id: inputCombo
+        model: inputComboModel
+        currentIndex: virtualstudio.inputDevice
+        onActivated: { virtualstudio.inputDevice = currentIndex }
+        x: 234 * virtualstudio.uiScale; y: virtualstudio.uiScale * (virtualstudio.selectableBackend ? 248 : 200)
+        width: parent.width - x - (16 * virtualstudio.uiScale); height: 36 * virtualstudio.uiScale
+        visible: virtualstudio.audioBackend != "JACK"
+    }
+    
+    ComboBox {
+        id: outputCombo
+        model: outputComboModel
+        currentIndex: virtualstudio.outputDevice
+        onActivated: { virtualstudio.outputDevice = currentIndex }
+        x: backendCombo.x; y: inputCombo.y + (48 * virtualstudio.uiScale)
+        width: backendCombo.width; height: backendCombo.height
+        visible: virtualstudio.audioBackend != "JACK"
+    }
+    
+    Text {
+        anchors.verticalCenter: inputCombo.verticalCenter
+        x: leftMargin * virtualstudio.uiScale
+        text: "Input Device"
+        font { family: "Poppins"; pixelSize: fontMedium * virtualstudio.fontScale * virtualstudio.uiScale }
+        visible: virtualstudio.audioBackend != "JACK"
+    }
+    
+    Text {
+        anchors.verticalCenter: outputCombo.verticalCenter
+        x: leftMargin * virtualstudio.uiScale
+        text: "Output Device"
+        font { family: "Poppins"; pixelSize: 13 * virtualstudio.fontScale * virtualstudio.uiScale }
+        visible: virtualstudio.audioBackend != "JACK"
+    }
+
+    Button {
+        id: refreshButton
+        background: Rectangle {
+            radius: 6 * virtualstudio.uiScale
+            color: refreshButton.down ? "#DEE0E0" : (refreshButton.hovered ? "#D3D4D4" : "#EAECEC")
+            border.width: 1
+            border.color: refreshButton.down || refreshButton.hovered ? "#BABCBC" : "#34979797"
```

Comment:
These shorts of sections should really be taken from the dark mode branch so that they don't need reworking later.